kuzzleio/documentation
# KZL-1212-add-banner-with-external-resources

[KZL-1212](https://jira.kaliop.net/browse/KZL-1212)

## What does this PR do?
Replace ``` Report an error on this page   | Request new content for this page``` links by banner with external ressources links.

PC
![image](https://user-images.githubusercontent.com/44427849/61222293-b1790f00-a71a-11e9-892f-1c4171c7d652.png)


Tablet
![image](https://user-images.githubusercontent.com/44427849/61222011-1122ea80-a71a-11e9-8427-6edee48c6b7e.png)

Mobile
![image](https://user-images.githubusercontent.com/44427849/61222048-25ff7e00-a71a-11e9-9e4e-df592edb81cf.png)

![image](https://user-images.githubusercontent.com/44427849/61222618-57c51480-a71b-11e9-81fe-0b47efef6071.png)


### How should this be manually tested?
  - Step 1 : npm run dev
  - Step 2 : check footer
  - Step 3 : verify banner links

### Other changes
/

### Boyscout
/
